orm/entities/TallySheetVersionRow/TallySheetVersionRow_PRE_21.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy.orm import relationship
+
+from app import db
+
+from orm.entities import Candidate
+from orm.entities.Election import ElectionCandidate
+from exception import NotFoundException
+from orm.entities.SubmissionVersion import TallySheetVersion
+
+
+class TallySheetVersionRow_PRE_21_Model(db.Model):
+    __tablename__ = 'tallySheetVersionRow_PRE_21'
+    tallySheetVersionRowId = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    tallySheetVersionId = db.Column(db.Integer, db.ForeignKey(TallySheetVersion.Model.__table__.c.tallySheetVersionId),
```

Comment:
This shouldnt be a primary key... I just figured the same mistake is there in PRE-41 too